stock_picking_group_by_partner_by_carrier_by_date/tests/test_grouping_by_date.py

```diff
@@ -56,17 +56,17 @@ def test_group_by_date(self):
             p1, p2 = self._create_orders_and_pickings()
             self.assertEqual(p1, p2)
 
-        # Same day, inside time window
+        # Same day
         with freeze_time("2020-11-27 11:00:00"):
             p3, p4 = self._create_orders_and_pickings()
             self.assertEqual(p3, p4)
             self.assertEqual(p3, p1)
 
-        # Same day, out of time window
-        with freeze_time("2020-11-27 18:00:00"):
+        # Same day, one second later
```

Comment:
```suggestion
        # Same day, one minute later
```
Or maybe change the frozen time to `2020-11-27 11:00:01` instead

Reply:
Fixed.